scripts/technical-debt-metrics.sh

```diff
@@ -46,26 +59,27 @@ computeDiff () {
 # the first `|` is a number.  This is then used for comparison and formatting.
 tdc () {
 titlesAndRegexes=(
```

Comment:
```suggestion
titlesPathsAndRegexes=(
```
Would you mind renaming this variable to something tracking also the middle entry?